product_docs/docs/pgd/5/raft/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+
+title: Proxies, Raft and Raft subgroups
+
+---
+
+PGD is based on a global Raft model where clusters have a single Raft group spanning all the data nodes in the PGD cluster. A Raft leader is elected by the cluster and propagates the state of the cluster to all the other nodes in the cluster. 
+
+!!! Hint What is Raft?
+Raft is an industry accepted algorithm for making decisions though achieving “consensus” from a group of separate nodes within a distributed system. 
+!!!
+
+It is essential for certain operations in the cluster that the Raft leader – such as adding and removing nodes and allocating ranges for [galloc](sequences/#pdg-global-sequences) sequences – is both established and maintained. 
+
+It also means that one half of the nodes in the cluster, plus one additional node must be able to reach each other. So, in a global cluster with five nodes, at least three of the nodes must be reachable by each other to establish a Raft leader.
+
+## Raft and proxy routing
+
+One function that also uses Raft is proxy routing. Proxy routing requires that the proxies are able to coordinate writing to a data node within their group of nodes. This data node is the write leader. If the write leader goes offline, the proxies need to be able to switch to a new write leader, selected by the data nodes, to maintain continuity for connected applications.
+
+Proxy routing can be configured in a number of ways in PGD 5, but the recommended configurations are "global" and "local" routing.
+
+Global routing uses the top-level group, also known as the global group, to manage the proxy routing. All nodes in the group are eligible to become write leader for all proxies. Connections to proxies within the global group will be routed to data nodes within the global group.
```

Comment:
This nicely shows our confused naming around top-level group, so we also call it global group, that's a 3rd name for it now :)

Reply:
Yes, the top-level group is global in TPA, so one has to mention both. Let me massage out the global group element.

Reply:
that's fine, I am just saying we need to figure something out, that works for everybody

Reply:
Change made in updates.